diff/cert.go

```diff
@@ -85,6 +85,18 @@ func (sc *Syncer) createUpdateCertificate(
 
 	// found, check if update needed
 	if !currentCertificate.EqualWithOpts(certificateCopy, false, true) {
+		currentSNIs, err := sc.currentState.SNIs.GetAllByCertID(*currentCertificate.ID)
```

Comment:
This is a fairly special case. Do you mind copying the git commit body in here as a code comment to help future maintainers?